src/components/context/startContext/ApplicationsPopup.tsx

```diff
@@ -80,7 +80,7 @@ export default function ApplicationsPopup({
       }
     };
     setApplications();
-  }, []);
```

Comment:
Has this been tested? Even when it says "missing dependency" adding it might have unwanted behaviour

Reply:
then adding ignore is the option as it is intentionally